### PR TITLE
refactor: signalling proposal subgraph change

### PR DESCRIPTION
**Pull request by Michael-A-Heuer**
## Description

Task ID: [OS-855](https://aragonassociation.atlassian.net/browse/OS-855)

Test subgraph: https://subgraph.satsuma-prod.com/aragon/osx-michael-baseGoerli/playground

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [x] I have updated the `UPDATE_CHECKLIST` file in the root folder.

[OS-855]: https://aragonassociation.atlassian.net/browse/OS-855?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/subgraph/schema.graphql [hunk 1]
```diff
@@ -461,7 +461,8 @@ type TokenVotingProposal implements IProposal @entity {
   startDate: BigInt!
   endDate: BigInt!
   creationBlockNumber: BigInt!
-  potentiallyExecutable: Boolean!
+  approvalReached: Boolean!
+isSignaling: Boolean!
```
Comment (Rekard0) [inline packages/subgraph/schema.graphql:465]:
```suggestion
  isSignaling: Boolean!
```

Comment (Michael-A-Heuer) [reply packages/subgraph/schema.graphql:465]: Rename to `isSignalling`

File: packages/subgraph/schema.graphql [hunk 2]
```diff
@@ -536,7 +537,8 @@ type AddresslistVotingProposal implements IProposal @entity {
...
```
Comment (Rekard0) [inline packages/subgraph/schema.graphql:541]:
```suggestion
  isSignaling: Boolean!
```

**Review by Rekard0 — DISMISSED** [review]
LGTM! apart from the minor suggestion

File: packages/subgraph/schema.graphql (hunk 2 above)
Comment (Michael-A-Heuer) [reply packages/subgraph/schema.graphql]: Fixed in 42ae70adc0882ca90e4b6a45c6a0d7bb1df1ebab

**Review by Rekard0 — APPROVED** [review]

**Review by josemarinas — APPROVED** [review]

**Pull request merged**